federation-2/router-bridge/src/planner.rs

```diff
@@ -1680,4 +1680,145 @@ feature https://specs.apollo.dev/unsupported-feature/v0.1 is for: SECURITY but i
 
         assert_eq!(expected.to_string(), error_to_display.to_string());
     }
+
+    #[tokio::test]
+    async fn deno_must_support_crypto_get_random_values() {
```

Comment:
This'll work but it seems like testing it via a bit of indirection.  I wonder if we should just have a small test in `js.rs` that literally just tests the JS runtime and a specific test that literally calls `crypto.getRandomValues`.

At the very least, please put a comment on here that says why we're testing this (e.g., because of a transitive dependency on `crypto.` via `uuid`, via , etc. — otherwise I don't think we'll understand why crypto is even in the picture.

Reply:
fixed in 4c31b6b